.github/workflows/changelog.yml

```diff
@@ -55,10 +47,11 @@ jobs:
               - '.github/actions/changelog/dist/**'
 
       - name: Get latest release
-        uses: actions-ecosystem/action-get-latest-tag@v1
         id: get-latest-tag
-        with:
-          semver_only: true
+        run: |
+          latest_tag=$(git describe --abbrev=0)
+          echo "::set-output name=tag::${latest_tag}"
+
```

Comment:
let's not change this for now. `action-get-latest-tag` just does a `git describe` anyway. we'll fix it in #1119